test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/ODataT4CodeGeneratorTests.cs

```diff
@@ -113,6 +131,17 @@ public void CodeGenSimpleEdmx()
             ODataT4CodeGeneratorTestDescriptors.Simple.Verify(code, false/*isCSharp*/, true/*useDSC*/);
         }
 
+        [TestMethod]
+        public void CodeGenSimpleEdmxMultipleFiles()
+        {
+            SplitFiles = true;
+            CompileGeneratedCode = false;

```

Comment:
What's the purpose of setting `CompilerGeneratedCode` to `false` here?

Reply:
At this point I wanted to content of the file without compiling the code.